scripts/dev_db_initialise.py

```diff
@@ -1,7 +1,7 @@
 from app import create_app
 from app.teams.service import create_team_in_db
 from app.users.service import create_user_in_db
-from app.team_members.service import create_team_member_in_db
+from app.team_members.service import create_membership_in_db
```

Comment:
This app.team_members.service path shouldn't exist? seems like it was changed before renaming the python module. would've thought it would make the unittests fail though?

Reply:
actually, the unittests make a test_db, rather than a dev db. that'll be why it passed unittests

Reply:
That script is used to generate some default data for testing out the app. The fact that this managed to make it in to the PR means it needs unit testing

Reply:
> 
> 
> actually, the unittests make a test_db, rather than a dev db. that'll be why it passed unittests

Oh, yeah, I just looked at the commit message it was added under. I clearly didn't really look at the file properly, I was focussing too much on your changes. unit tests would be good

Reply:
It was a good find, I'll get this sorted tonight